paper/paper.md

```diff
@@ -39,29 +39,28 @@ affiliations:
-Using alignments to reference chloroplast sequences and the k-mer distribution candidate chloroplast reads are extracted and assembled.
-This targeted assembly is much faster and yields less contigs compared to an assembly of all reads.
+Using alignments to reference chloroplast sequences and the k-mer distribution candidate chloroplast reads are extracted from the complete set.
+Afterwards, the targeted assembly of those sequences is much faster and yields less contigs compared to an assembly of all reads.
 Assemblers usually fail to assemble chloroplast genomes as a single contig due to their structure, consisting of two single copy regions and an inverted repeat.
 The size of the inverted repeat is in most cases multiple kilobasepairs in size, therefore it can not be resolved using short reads only.
-However SPAdes (cite) returns the assembly graph where the typical chloroplast structure can be recognized and reconstructed using the knowledge of its structure.
-This leads to a full chloroplast assembly as a single contig within minutes on this demo set: SRRxxxxxxx
-
-Whole chloroplast genomes are useful in ...
-The chloroExtractor bridges the gap between sequencing and annotation tools like DOGMA (cite), cpGAWAS (cite) and VERDANT (cite).
-We plan to use the chloroExtractor also to screen SRA for chloroplast genomes in public datasets that are not yet available in chloroplast databases (cite NCBI and chloroDB).
-The extraction of chloroplast sequences can be useful in order to remove chloroplast reads for the genomic assembly.
-
-A similar tool to take note of is orgASM a dedicated organelle assembler written in python.
-Also plasmid SPAdes could possibly be used for this purpose although it is not intended for it.
+However SPAdes [@Spades_2013] returns the assembly graph where the typical chloroplast structure can be recognized and reconstructed using the knowledge of its structure.
+Using our demo set, one can achieve a single contig assembly of the chloroplast of *Spinacia oleracea* .
+The final chloroplast sequence can be further annotated with tools like DOGMA [@dogma2004], cpGAVAS [@Liu2012] and VERDANT [@Mckain2017].
+Such assemblies, can be used to remove chloroplast reads before a genomic assembly of the remaining nuclear DNA.
+Moreover, chloroplast genomes are useful in phylogenetic reconstruction [@huang_2016] or barcoding applications [@Coissac_2016].
+A similar tool, aiming the assembly of whole chloroplast genomes is the Python program org.ASM [https://git.metabarcoding.org/org-asm/org-asm], but it is not production ready, yet.
+Also plasmid SPAdes [@plasmidspades_2016] could possibly be used for this purpose although it is not intended for it.
+In the future, we plan to use our chloroExtractor to screen NCBI's Sequence Read Archive [@sra2008] for chloroplast genomes in public sequencing datasets that are not yet available in chloroplast databases, eg. chloroDB [@chlordb2006] to broaden our knowledge about chloroplasts.
```

Comment:
This reference is to NCBI as a whole. That is not a bad idea, but maybe we should add the more specific citation for SRA: https://doi.org/10.1093/nar/gkq1019

Reply:
okay... Will add that, but we will remove the citation in paper.bib with your issue above?